libs/solid/quiz/src/lib/components/drag-and-drop-choice-question/drag-and-drop-question.component.ts

```diff
@@ -1,18 +1,96 @@
-import { Component, Input } from '@angular/core';
-import { DragAndDropQuestion } from '../../state/quiz.model';
+import { Component, Input, OnInit } from '@angular/core';
+import {CdkDragDrop, moveItemInArray} from '@angular/cdk/drag-drop';
+import { DragAndDropQuestion, QuizAnswer } from '../../state/quiz.model';
 import { QuestionTypeComponent } from '../question/question.component';
 
+
+class Status {
+  constructor(public text: string, public status: string) {
+  }
+}
+
 @Component({
   selector: 'solid-quiz-drag-and-drop-question',
   templateUrl: './drag-and-drop-question.component.html',
   styleUrls: ['./drag-and-drop-question.component.scss'],
 })
-export class DragAndDropQuestionComponent implements QuestionTypeComponent {
+export class DragAndDropQuestionComponent implements QuestionTypeComponent, OnInit {
   @Input() public question!: DragAndDropQuestion;
   @Input() public showAnswers = false;
+  isShow: Boolean = true;
+
+  status: Status = new Status('', '');
+  result: Status[] = [];
+  movies = [
+    ' I - The Phantom Menace',
+    ' II - Attack of the Clones',
+    'III - Revenge of the Sith',
+    'IV - A New Hope',
+    'V - The Empire Strikes Back',
+  ];
+
+  duplicate = [
+    ' I - The Phantom Menace',
+    ' II - Attack of the Clones',
+    'III - Revenge of the Sith',
+    'IV - A New Hope',
+    'V - The Empire Strikes Back',
+  ];
+
+  ngOnInit(): void {
+    console.log(this.status);
+    this.movies = this.shuffle(this.movies);
+  }
+
+  public trackByFn(item: QuizAnswer) {
+    return item.id;
+  }
+
+  shuffle(array: any[]) {
```

Comment:
Currently this implementation can "shuffle" the correct order. Please add an check to prevent this.